### PR TITLE
Add a don't to Do's and Don'ts list

### PR DESCRIPTION
**Pull request by Mwalters75**
Adds a don't to not install a Wii Startup Disc WAD to your Wii, as it would brick it.

### CODE REVIEW

File: _pages/en_US/dosanddonts.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ Here's a list of things you should and should not do once you've modded your Wii
 - **DO NOT** use the homebrew app "KoreanKii" on a non-Korean Wii.
 - **DO NOT** Install any IOS or Wii Menu on a Wii mini other than [d2xl cIOS](cios-mini). Doing so will brick your console if a Wi-Fi card is not soldered to it.
 - **DO NOT** use any Nintendo Wi-Fi Connection replacement on your Wii except for [Wiimmfi](wiimmfi). Unfortunately, other replacement servers are vulnerable to an exploit that can brick your Wii if you use them. [RiiConnect24](riiconnect24) is safe to use as well.
+- **DO NOT** install a Wii Startup Disc WAD to your Wii.
```
Comment (osfanbuff63) [inline _pages/en_US/dosanddonts.md:23]:
```suggestion
- **DO NOT** install a Wii Startup Disc WAD to your Wii. Doing so will brick your Wii!
```

Comment (Mwalters75) [reply _pages/en_US/dosanddonts.md]: Done.

Comment (osfanbuff63) [reply _pages/en_US/dosanddonts.md]: I guess, but I was taking from other parts of this page (and the fact that we have this page makes it seem like it should be as clear as possible. New people to homebrew might not know as well, esp with the startup disc. On the other point, maybe someone thinks "oh, I found a channel that launches the old startup disc! I'll go install that for nostalgia maybe..."

Comment (Mwalters75) [reply _pages/en_US/dosanddonts.md]: > I guess, but I was taking from other parts of this page (and the fact that we have this page makes it seem like it should be as clear as possible. New people to homebrew might not know as well, esp with the startup disc. On the other point, maybe someone thinks "oh, I found a channel that launches the old startup disc! I'll go install that for nostalgia maybe..."

Someone on the RC24 Discord found a WAD of it, installed it, and the video doesn't output

Comment (osfanbuff63) [reply _pages/en_US/dosanddonts.md]: Yeah, maybe something like `Do NOT install any random WADs downloaded off the Internet! <list of wads that are safe: e.g. rc24 wads, wiilink24 wads, etc>` would be better.

File: _pages/en_US/dosanddonts.md [hunk 2]
```diff
@@ -20,6 +20,7 @@ Here's a list of things you should and should not do once you've modded your Wii
 - **DO NOT** use the homebrew app "KoreanKii" on a non-Korean Wii.
 - **DO NOT** Install any IOS or Wii Menu on a Wii mini other than [d2xl cIOS](cios-mini). Doing so will brick your console if a Wi-Fi card is not soldered to it.
 - **DO NOT** use any Nintendo Wi-Fi Connection replacement on your Wii except for [Wiimmfi](wiimmfi). Unfortunately, other replacement servers are vulnerable to an exploit that can brick your Wii if you use them. [RiiConnect24](riiconnect24) is safe to use as well.
+- **DO NOT** install a Wii Startup Disc WAD to your Wii. Doing so will brick your Wii!
```
Comment (osfanbuff63) [inline _pages/en_US/dosanddonts.md:23]: From the input on the review thread above (thanks Peter0x44), are there any others that should be mentioned?
```suggestion
- **DO NOT** install random WAD files from the internet! However, WADs downloaded from the patchers of [RiiConnect24](riiconnect24) or [WiiLink24](wiilink) are known to be safe.
```

**Review by osfanbuff63 — CHANGES_REQUESTED** [review]

**Comment by larsenv** [comment]
This is gonna mislead people.

**Pull request closed without merging**